evaluator/autoware_planning_evaluator/include/autoware/planning_evaluator/metrics/metrics_utils.hpp

```diff
@@ -14,7 +14,6 @@
 
 #ifndef AUTOWARE__PLANNING_EVALUATOR__METRICS__METRICS_UTILS_HPP_
 #define AUTOWARE__PLANNING_EVALUATOR__METRICS__METRICS_UTILS_HPP_
-
```

Comment:
remove diff

Reply:
adressed in 104fb0601b56d4666befa72ae147a171ec36e7f6